components/02-content/carousel/carousel.hbs

```diff
@@ -17,11 +17,11 @@
 
   {{! Controls }}
   <a href="#{{ carouselId }}" class="left carousel-control" role="button" data-slide="prev">
-    {{> '@icon' icon="glyphicon glyphicon-chevron-left" }}
+    {{> '@icon' icon='chevron-left icon-prev' }}
```

Comment:
Waarom hier 2 classes? Kan dat niet ineen?

Reply:
Dat zou nu best nog wel eens kunnen.. Als ik er tijd voor heb fiets ik ze om

Reply:
Scheelt ook in je releasenotes